README.md

```diff
@@ -76,6 +76,26 @@ To migrate from S3 to local storage, the identified steps will be:
     `bin/rake cache:clear`
 5. Restart the Rails server
 
+### Detect spam
+
+To detect spam in Decidim.
+
+#### Detect spam users
+Detects users susceptible of being spammers. It can run on all organizations or be scoped to a single organization by passing the organization ID as the rake task parameter.
+
+This rake task export a .csv with a list of all the searched users. A column indicates if each user is suspicious of being a spammer or not.
+The columns in the CSV are: "ID, "Is suspicious?", "Name", "Email", "Nickname", "Personal URL", "About"
+
+Examples:
+`bin/rake cdtb:spam:users[org_id]` --> find users in organization with an id.
+`bin/rake cdtb:spam:users` --> find all users in all organizations.
+
+To set custom words in the rake, you can override it with an initalizer:
+
+Decidim::Cdtb.configure do |config|
+  config.spam_words = ENV["CDTB_SPAM_WORDS"]&.split(",")
+end
```

Comment:
Good! le'ts make it more readable
````suggestion
```
Decidim::Cdtb.configure do |config|
  config.spam_words = ENV["CDTB_SPAM_WORDS"]&.split(",")
end
```
````

Reply:
the suggestion is the same